Source/controls/modifier_hints.cpp

```diff
@@ -114,8 +114,8 @@ void DrawStartModifierMenu(const CelOutputBuffer &out)
 {
 	if (!start_modifier_active)
 		return;
-	static const CircleMenuHint dPad(/*is_dpad=*/true, /*top=*/N_("Menu"), /*right=*/N_("Inv"), /*bottom=*/N_("Map"), /*left=*/N_("Char"));
-	static const CircleMenuHint buttons(/*is_dpad=*/false, /*top=*/"", /*right=*/"", /*bottom=*/N_("Spells"), /*left=*/N_("Quests"));
+	static const CircleMenuHint dPad(/*is_dpad=*/true, /*top=*/_("Menu"), /*right=*/_("Inv"), /*bottom=*/_("Map"), /*left=*/_("Char"));
+	static const CircleMenuHint buttons(/*is_dpad=*/false, /*top=*/"", /*right=*/"", /*bottom=*/_("Spells"), /*left=*/_("Quests"));

```

Comment:
If this works then we need to remove the additional translation from `Source/controls/modifier_hints.cpp::33-40`

Reply:
Yes it does :)
I've done those change before the add of the translation line33-40.
I've re-check and the hints are properly tranlated into the bellow screenshot

![Capture d’écran du 2021-05-01 14-29-31](https://user-images.githubusercontent.com/148688/116782735-2aca2900-aa8b-11eb-8492-eda6fc3ca413.png)